lib/radiator/auth/permission.ex

```diff
@@ -0,0 +1,124 @@
+defmodule Radiator.Auth.Permission do
```

Comment:
Fair point. I remember a context rule from the guides that says, if you have logic that deals with stuff from two context, add a new context at the intersection. Maybe that's a way here? Rename `Radiator.Auth` to `Radiator.Authentication` and add `Radiator.Authorization` for permission concerns?

Reply:
Okay, generally speaking naming wise I wanted to go with Auth - for Authentication and Perm/Permissions instead of Authorization, so we don't have that annoying name prefix match. 

But then again in that thinking neither `.Auth` nor `.Perm` represent a context. So for our currentpurposes I probably would recommend putting the Authorization part into a internal context, living in `Directory.Editor.Permissions` - and we probably need to abstract it more so another plugin could add their permissions as needed, e.g. eventually design something that can be extended by behaviors.

So we have `.Auth` and `.Perm` in which content independed base structures, types and functions live that any Radiator based application could reuse. And we do have `Directory.Editor.Permissions` wherein the "base" Radiator Directory functionality of permissions lives. Does that make sense to you?